tests/framework/utils/helm_helper.go

```diff
@@ -18,11 +18,12 @@ package utils
 
 import (
 	"fmt"
-	"gopkg.in/yaml.v2"
```

Comment:
Could this file be reverted? It looks like nothing really changed and not related to the PR?

Reply:
done